homeassistant/components/bond/light.py

```diff
@@ -231,3 +285,31 @@ async def async_turn_off(self, **kwargs: Any) -> None:
         _LOGGER.debug("Fireplace async_turn_off called with: %s", kwargs)
 
         await self._hub.bond.action(self._device.device_id, Action.turn_off())
+
+    async def async_set_brightness_belief(self, brightness: int) -> None:
+        """Set the belief state of the light."""
+        if not self._device.supports_set_brightness():
+            raise HomeAssistantError("This device does not support setting brightness")
+        if brightness == 0:
+            await self.async_set_power_belief(False)
```

Comment:
```suggestion
            await self.async_set_power_belief(False)
            return
```

Then remove the else and outdent below